streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -387,28 +469,34 @@ public String queryableStoreName() {
         return toStream().selectKey(mapper);
     }
 
+    @Override
+    public <K1> KStream<K1, V> toStream(final KeyValueMapper<? super K, ? super V, ? extends K1> mapper,
+                                        final Named named) {
+        return toStream(named).selectKey(mapper);
+    }
+
     @Override
     public KTable<K, V> suppress(final Suppressed<? super K> suppressed) {
         final String name;
+        final NamedInternal renamed;
```

Comment:
A while back there was a discussion on how to handle naming `suppression` and it's a different case.  We only want to invoke the `builder.newProcessorName` in cases where the user has not supplied a name.  So the code for suppress should stay the same.  

Although we might lose context as to why this is different, I believe the Javadoc contains the explanation so we should be covered.

\cc @vvcephei

Reply:
Yep. Thanks, @bbejeck !